src/Sql/NotificationCenter/dbo/Tables/Notification.sql

```diff
@@ -0,0 +1,32 @@
+CREATE TABLE [dbo].[Notification]
+(
+    [Id] UNIQUEIDENTIFIER NOT NULL,
+    [Priority] TINYINT NOT NULL,
+    [Global] BIT NOT NULL,
+    [ClientType] TINYINT NOT NULL,
+    [UserId] UNIQUEIDENTIFIER NULL,
+    [OrganizationId] UNIQUEIDENTIFIER NULL,
+    [Title] NVARCHAR (MAX) NULL,
```

Comment:
🎨 This should probably be 255 or 256 or whatever is the norm here.

Reply:
Settled on 256. Fixed in 006f1b4 and 78058d9